### ISSUE TITLE
Developers Account link is not working

### ISSUE THREAD

**Issue by SabarishT** [ambrosus/sdk-javascript#25]
Developers Account link
[](https://selfservice-test.ambrosus.com/create) is not working. Please anybody tell me updated link to create address and secret.

**Comment by SabarishT** [comment]
Thank you guys. I finally got developer's account updated link
[https://dashboard.hermes.ambrosus-test.com/signup](url)

**Comment by manohar240** [comment]
None of the below links are working for registration.
[https://selfservice-test.ambrosus.com/create](https://selfservice-test.ambrosus.com/create)
[https://dashboard.hermes.ambrosus-test.com/signup](https://dashboard.hermes.ambrosus-test.com/signup)

Request you to please share new working link.

Request you to please share new working link.

**Comment by SabarishT** [comment]
Yes, They are not working right now because they are updating their current system. Currently there are no working public test environments. Maybe They will update in near future. So please contact ambrosus team regarding this.
